lib/modules/manager/git-submodules/extract.spec.ts

```diff
@@ -1,69 +1,89 @@
 import is from '@sindresorhus/is';
 import { mock } from 'jest-mock-extended';
-import _simpleGit, {
-  Response,
-  SimpleGit,
-  SimpleGitFactory,
-  TaskOptions,
-} from 'simple-git';
+import { Response, SimpleGit, SimpleGitFactory, simpleGit } from 'simple-git';
 import { GlobalConfig } from '../../../config/global';
 import * as hostRules from '../../../util/host-rules';
-import type { PackageFileContent } from '../types';
 import { extractPackageFile } from '.';
 
 jest.mock('simple-git');
-const simpleGit: jest.Mock<Partial<SimpleGit>> = _simpleGit as never;
+const simpleGitFactoryMock = simpleGit as jest.Mock<Partial<SimpleGit>>;
```

Comment:
I think we've some helper functions inside test utils for easier handling.

Reply:
I looked at them and tried to use them, without luck. If you have a concrete implementation idea, happy about using that. Otherwise I think the current solution can be acceptable.